host/xtest/xtest_benchmark_1000.c

```diff
@@ -0,0 +1,174 @@
+/*
+ * Copyright (c) 2015, Linaro Limited
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License Version 2 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ */
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+#include "xtest_test.h"
+#include "xtest_helpers.h"
+
+#include <ta_storage_benchmark.h>
+
+#define DO_VERIFY 0
+#define DEFAULT_DATA_SIZE (2 * 1024 * 1024) // 2MB
+#define DEFAULT_CHUNK_SIZE (1 * 1024) // 1KB
+#define DEFAULT_COUNT (10)
+
+size_t data_size_table[] = {
+	256,
+	512,
+	1024,
+	2 * 1024,
+	4 * 1024,
+	16 * 1024,
+	512 * 1024,
+	1024 * 1024,
+	0
+};
+
+#define ARRAY_SIZE(A) (sizeof(A)/sizeof(*(A)))
```

Comment:
We have this in for example xtest_20000.c also. When starting to duplicating common macro's, we should put them in a shared h-file instead.